GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Agena/RO_Agena.cfg

```diff
@@ -0,0 +1,797 @@
+	@mass = 0.09
+	@maxTemp = 773.15
+	@skinMaxTemp = 873.15
+	@attachRules = 1,1,1,1,0
+	!RESOURCE,* {}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		type = ServiceModule
+		volume = 30
+		basemass = -1
+	}
+	!MODULE[ModuleB9PartSwitch]:HAS[#moduleID[meshSwitchTanks]] {}
+	MODULE
+	{
+		name = ModuleB9PartSwitch
+		moduleID = meshSwitchTanks
+		switcherDescription = Tank
+		switcherDescriptionPlural = Tanks
+		affectDragCubes = True
+		affectFARVoxels = True
+		SUBTYPE
+		{
+			name = Two
+			transform = Mesh_TwoBall
+		}
+
+		SUBTYPE
+		{
+			name = Three
+			transform = Mesh_ThreeBall
+			MODULE
+			{
+				IDENTIFIER { name = ModuleFuelTanks }
+				DATA
+				{
+					volume = 45
+				}
+			}
+		}
+	}
+	%useRcsConfig = RCSBlockTenth
+	!MODULE[ModuleResourceConverter] {}
+}
+
+
+// Horizon Sensor
+@PART[bluedog_Agena_HorizonSensor]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.6
+	@title = Agena Infrared Horizon Sensor Module
+	@manufacturer = Barnes Engineering
+	@description  = A small horizon scanner with a jettisonable fairing intended for the Agena upper stage. Used as one unit on the Agena A and B upper stage tanks and used in pairs on the Agena D upper stage telemetry unit. On Agena D, the each horizon sensor is 35 degrees apart from the center of the back face, and on the Agena Target Vehicle they are placed on the Agena Target Vehicle Materials Bay each 45 degrees from the center. Provides SAS modes.
+	@mass = 0.01
+}
+
+
+// P-11
+@PART[bluedog_Agena_Subsat_Hitchhiker]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@title = P-11 Subsat
+	@manufacturer = Lockheed
+	@mass = 0.06
+	!RESOURCE,* {}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 2
+		type = PSPC
+		basemass = -1
+		TANK
+		{
+			name = PSPC
+			amount = 2
+			maxAmount = 2
+		}
+	}
+	MODULE
+	{
+		name = ModuleEngineConfigs
+		type = ModuleEngines
+		configuration = P-11
+		origMass = 0.06
+		CONFIG
+		{
+			name = P-11
+			minThrust = 0.0
+			maxThrust = 6.0
+			heatProduction = 100
+
+			PROPELLANT
+			{
+				name = PSPC
+				ratio = 1.0
+				DrawGauge = True
+			}
+			atmosphereCurve
+			{
+				key = 0 294
+				key = 1 100
+				key = 7 0.001
+			}
+		}
+	}
+	RESOURCE
+	{
+		name = HTP
+		amount = 10
+		maxAmount = 10
+	}
+	RESOURCE
+	{
+		name = Helium
+		amount = 112.5
+		maxAmount = 112.5
+	}
+	RESOURCE
+	{
+		name = ElectricCharge
+		amount = 1500
+		maxAmount = 1500
+	}
+	@MODULE[ModuleRCSFX]
+	{
+		@thrusterPower = 0.05
+		!resourceName = NULL
+		PROPELLANT
+		{
+			name = HTP
+			ratio = 1.0
+			DrawGauge = True
+		}
+		PROPELLANT
+		{
+			name = Helium
+			ratio = 11.25
+			ignoreForIsp = True
+		}
+	}
+}
+
+
+// SLV3B Interstage
+@PART[bluedog_Agena_SLV3B_Interstage]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@title = Agena SLV3B Interstage
+	@manufacturer = Convair
+	@description = Half length 1.5m interstage for the Atlas SLV-3B rocket.
+	@mass = 0.1

```

Comment:
Could you try to configure these a bit better instead of setting them all the same?